lib/matplotlib/cbook/deprecation.py

```diff
@@ -341,17 +343,23 @@ def func(used_arg, other_arg, unused, more_args): ...
         for param in signature.parameters.values()])
 
     @functools.wraps(func)
-    def wrapper(*args, **kwargs):
-        arguments = func.__signature__.bind(*args, **kwargs).arguments
+    def wrapper(*f_args, **f_kwargs):
+        arguments = func.__signature__.bind(*f_args, **f_kwargs).arguments
         # We cannot just check `name not in arguments` because the pyplot
         # wrappers always pass all arguments explicitly.
-        if name in arguments and arguments[name] != _deprecated_parameter:
+        if any(name in d and d[name] != _deprecated_parameter
+               for d in [arguments, arguments.get(kw_name, {})]):
+            addendum = (
+                f"If any parameter follows {name!r}, they should be passed as "
+                f"keyword, not positionally.")
+            if kwargs.get("addendum"):
```

Comment:
AFAICS this is a new optional parameter of `_delete_parameter`. Please make it explicit.

Reply:
I now mentioned that all additional parameters are forwarded to warn_deprecated.